src/view/screens/Feeds.tsx

```diff
@@ -644,7 +644,7 @@ function SavedFeed({
     savedFeed.type === 'feed' ? savedFeed.view.displayName : savedFeed.view.name
 
   return (
-    <FeedCard.Link testID={`saved-feed-${feed.displayName}`} feed={feed}>
+    <FeedCard.Link testID={`saved-feed-${feed.displayName}`} {...savedFeed}>
```

Comment:
Nit: Spread props become regrettable over time. Let's do it explicitly unless it's like 10 fields.

Reply:
Oh yea, didn't like this either. Can you explain to me why the type is wonky though if I explicitly set `type` and `view` props

Reply:
Ok let's not screw around with types for now